javascript-light-bulb/main.js

```diff
@@ -0,0 +1,22 @@
+const $lightblub = document.querySelector('.light');
+const $container = document.querySelector('.container');
+
+let on = true;
+
+function counter(event) {
+  if (on === true) {
+    $lightblub.className = 'light shadow sunny';
+    $container.className = 'container day';
+    console.log(on);
+    on = false;
+    console.log($lightblub.className);
+  } else if (on === false) {
+    $lightblub.className = 'light shadow moony';
+    $container.className = 'container night';
+    console.log(on);
+    on = true;
+    console.log($lightblub.className);
+  }
+}
```

Comment:
Remove all `console.log`s before submitting your code. They are only for testing / debugging purposes